### PR TITLE
API v2: Allow array as name servers update payload

### PR DESCRIPTION
**Pull request by jodosha**

### CODE REVIEW

File: content/v2/domains/name-servers.markdown [hunk 1]
```diff
@@ -74,7 +74,21 @@ Name | Type | Description
 ##### Example
 
 ~~~json
-{}
+"{
+  "name_servers": [
+    "ns1.example.com",
+    "ns2.example.com"
+  ]
+}"
```
Comment (weppos) [inline content/v2/domains/name-servers.markdown]: @jodosha I think we should just support one way, unless it proves to be confusing. I'd only document (and support) the array version.

**Comment by weppos** [comment]
@jodosha any proposal on how to update the following section?

```
Name | Type | Description
-----|------|------------
`name_servers` | `array` | **Required** A list of name server names as strings.
```

**Comment by jodosha** [comment]
@weppos Should it be unnamed?

**Comment by weppos** [comment]
I wonder if we should replace the entire section with

```
**Required** A list of name server names as a JSON-encoded array of strings.
```

**Comment by jodosha** [comment]
@weppos That would lack of consistence with other payload descriptions. My suggestion is:

```
```
Name | Type | Description
-----|------|------------
 | `array` | **Required** A list of name server names as strings.
```
```

**Comment by weppos** [comment]
I thought Markdown would complain about a table with an empty content. I'm fine to go that way.

**Comment by jodosha** [comment]
@weppos There is an empty space, btw :wink:

**Comment by jodosha** [comment]
screen shot 2016-01-29 at 15 17 34

This is the rendering.

**Comment by weppos** [comment]
:+1: deployed

**Pull request merged**